### DISCUSSION TITLE
How can I connect Supabase Self-Hosted to an external database?

### DISCUSSION THREAD

**Discussion by billypap1** [supabase/supabase#7018] [Questions] [answered]
I have self-hosted Supabase but there are no docs in order to connect supabase to an external Postgres Database instead of the built in.

**Comment by Nemmcy** [comment]
Found an answer?

**Comment by khanakia** [comment]
I am looking for a solution too

**Reply by vogelcodes** [reply]
Have you managed to get your JWT secrets working? I'll connect it to a external postgres later. Maybe we can help each other out.

**Comment by vogelcodes** [comment, marked as answer]
I think you need to update the values in .env to point to the right host. There's a supabase/postgres image that you can host somewhere else.

**Reply by J0** [reply]
Hey @vogelcodes,

Thanks for the typing out a solution. The `.env` variables, specifically the `POSTGRES_*` variables need to be updated in order to use an external database. For further reference, feel free to take a look at [this guide](https://supabase.com/docs/guides/hosting/docker#using-an-external-database)

**Reply by flavienbwk** [reply]
@J0 Do you have any tutorial explaining how to configure the Supabase CLI so it can be used with this Docker configuration ?
